README.md

```diff
@@ -58,58 +65,107 @@ To apply this template to an existing repository, follow these steps:
-1. Download the zip file of this repository and extract it to a temporary location.
-1. Copy the contents from the extracted folder to the root of your repository, with the exception of the `custom` folder (you should already have a folder for the module you're working with).
-    - Depending on what you already have, this will overwrite some files in your repository, and it will discard some configuration you might want to keep, but we'll review those in a later step and recover them if necessary using git.
-1. Review and update the `.iqgeorc.jsonc` file to match your project settings and dependencies.
-1. Ensure you have the IQGeo VSCode extension installed. It's available in the Extensions Marketplace.
+2. Apply the template to your repository:
+    - If you have the IQGeo VSCode extension installed, you can use the command "IQGeo Pull and merge files from project-template" to apply the template to your repository.
+    - If you don't have the extension installed, you can download the zip file of this repository and extract it to a temporary location. Then copy the contents from the extracted folder to the root of your repository, with the exception of the `custom` folder (you should already have a folder for the module you're working with).
+1. Review changes. Depending on what you already have, this will overwrite some files in your repository, and it will discard some configuration you might want to keep, but we'll review those in a later step and recover them if necessary using git.
+1. Follow the steps in following section [Configuring the project](#configuring-the-project) to configure the project.
+
+
+## Configuring the project
+
+To configure the project you'll review and update the `.iqgeorc.jsonc` file to match your project settings and dependencies. 
+First check the following section for additional requirements for some modules.
+
+
+### Notes on Product modules
+
+The `.iqgeorc.jsonc` provides a way to specify IQGeo products to be included in the project. This is done by adding entries to the `modules` array. The file includes comments detailing the options that can be set when specifying a module.
+
+Some modules will have additional requirements, see the following list for the requirements of each module:
+
+#### comsof:
+
+- requires `comms` module to be included in the project as well
+
+#### comms_dev_db:
+
+- to be used in dev environments only.  include `"devOnly": true` in the properties for this module in the `.iqgeorc.jsonc` file
+- requires both `comms` and `comsof` modules to be included in the project as well
+- to create the db on startup of the container (if there's no db with NMT schema) replace the contents of the file `.devcontainer/entrypoint.d/600_init_db.sh` with the following:
+  
+      #!/bin/bash
+      if ! myw_db $MYW_DB_NAME list versions --layout keys | grep myw_comms_schema | grep version=; then $MODULES/comms_dev_db/utils/comms_build_dev_db --database $MYW_DB_NAME fi
+
```

Comment:
There is a syntax error in this line. Missing the ; before the fi.

Should be
`if ! myw_db $MYW_DB_NAME list versions --layout keys | grep myw_comms_schema | grep version=; then $MODULES/comms_dev_db/utils/comms_build_dev_db --database $MYW_DB_NAME; fi`